_node-machine_to_Syncano-script/generateCiCredentials.js

```diff
@@ -0,0 +1,16 @@
+const generateCiCredentials = () => {
+  const ciBaseEmail = process.env.CI_BASE_EMAIL;
```

Comment:
Add all necessary env variable to the readme, and also set it by default to `syncano.bot@`